charts/ztka/values.yaml

```diff
@@ -483,6 +483,9 @@ filebeat:
                       fields:
                         paralus.component: relay
                       processors:
+                        - drop_event:
+                            when:
+                              has_fields: ['json.caller']  # these will be application logs
```

Comment:
@meain  Do we need same setting for other templates?

Reply:
No, others are pretty much just running `tail -F` on a file which just contains audit logs. They too generate errors but since they are not parseable into json, they are skipped. This one is picking output from `relay` in tail mode which is why we have to ignore the application logs.